webroot/admin/user-mgmt.php

```diff
@@ -42,9 +43,18 @@
         echo "<td>" . $user->getFirstname() . " " . $user->getLastname() . "</td>";
         echo "<td>" . $user->getUID() . "</td>";
         echo "<td>" . $user->getOrg() . "</td>";
-        echo "<td><a href='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
+        echo "<td><a href ='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
```

Comment:
Getting rid of extra space on the a tag

```suggestion
        echo "<td><a href='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
```